dev/org.eclipse.codewind.openapi.core/build.properties

```diff
@@ -3,4 +3,5 @@ output.. = build/classes/
 bin.includes = META-INF/,\
                .,\
                .options,\
-               lib/
+               lib/,\
+               plugin.xml
```

Comment:
plugin.properties needs to be added here for it to show up in the packaged update site

Reply:
@sghung , good catch.  Change has been added.